src/components/postgresConnect.ts

```diff
@@ -663,6 +664,17 @@ class PostgresDB {
     }
   }
 
+  public getDistinctPermissions = async (isSystemOrgPermission: boolean = false) => {
+    const text = 'select * from permission_name where is_system_org_permission = $1'
```

Comment:
Would prefer keeping to SQL in CAPS as per most of the other queries, but not a big deal:
```
SELECT * FROM permission_name WHERE is_system_org_permission = $1'
```